spec/spec_helper.rb

```diff
@@ -101,13 +101,19 @@
 
   original_wd  = Dir.pwd
   original_env = ENV.to_hash.delete_if {|k, _v| k.start_with?(Bundler::EnvironmentPreserver::BUNDLER_PREFIX) }
+  original_default_specs = Dir[File.join(Gem.default_dir, "specifications", "default", "bundler*")]
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
   end
 
   config.before :all do
     build_repo1
+    original_default_specs.each {|s| FileUtils.mv(s, s + ".org") }
```

Comment:
why do we need this?

Reply:
some examples that use fake bundle commands are failing status when there is bundler as default gems.

see https://travis-ci.org/bundler/bundler/jobs/313289214